src/components/Garden/Feed/ProposalSupport.tsx

```diff
@@ -61,7 +75,18 @@ function DecisionSummaryBar({ proposal }) {
 
   const YouTag = (
     <div>
-      <Tag>You</Tag>
+      <Tag
+        background={undefined}
+        color={undefined}
+        limitDigits={undefined}
+        icon={undefined}
+        label={undefined}
+        mode={undefined}
+        size={undefined}
+        uppercase={undefined}
```

Comment:
Why its defining undefined here?

Reply:
Since this is a hive-ui component and it's not optional the props I did this. I don't like it also, but it was just a consequence of migrating this file to TS 😅

Reply:
Damn! I'll check with @reznord if we can get that types that i rewrite rolling, it should fix all that.